### ISSUE TITLE
Not allowed on home and login page

### ISSUE THREAD

**Issue by fabfurnari** [mickael-kerjean/filestash#127]
Hello, just downloaded the docker-compose file to run filestash on localhost.

If I access http://localhost:8334/ or http://localhost:8334/login I get

`{"status":"error","message":"Not Allowed"} `

If I access the /admin path I can configure everything and clicking on the logo image I get the actual /login page with forms for all services but every time I try to access any of those I get a "Not Allowed" popup box (I tried with Dropbx, SFTP, FTP).

If I use the "old" `machines/nuage` image everything works just fine.

Thanks

**Comment by mickael-kerjean** [comment]
Hello,

You should be able to solve the problem either by:
1. setting the [APPLICATION_URL](https://github.com/mickael-kerjean/filestash/blob/master/docker/prod/docker-compose.yml#L8) in your `docker-compose.yml` such as you have: "localhost:8334"
2. going into the admin console under `http://localhost:8334/admin` and fill what's under `general -> host` either with:
- "localhost:8334"
- nothing at all: however, this is not a great idea for any production instance as the error you see is a security feature to avoid anyone tampering. The idea being that any request that does not originate from your host will be rejected

**Comment by fabfurnari** [comment]
I can confirm this, setting the envvar `APPLICATION_URL` works. I tried before opening but I didn't set the port. Using the full url with port works fine. 

Thanks,

**Comment by SRugina** [comment]
Hi, one of my colleagues edited some things on the dashboard and then this error started occurring - so I decided to reinstall filestash, yet when I go to http://serverIP:8334/admin/setup I get this error. I tried doing `export APPLICATION_URL=localhost:8334` before running the file stash executable but still getting the same error. The config.json file in /app/data/config/config.json also has host set to localhost:8334 by default. Is there something glaringly obvious that I'm missing? 

Thanks in advance

**Comment by mickael-kerjean** [comment]
My little finger tells me you're using the master branch which hasn't been released to the docker hub yet because my CI server did his job and caught some errors while building and testing the last commit. A fix will come in soon, but if you need reliability and stability without using the official docker container, drop me an email and we can arrange a chat to go more in the details of how I can help you.

**Comment by mickael-kerjean** [comment]
@SRugina: the tests just passed and the build got deployed, problem solved

**Comment by thomasInes** [comment]
Hi,
 I have the same issue but I made sure the APPLICATION_URL var is set.
I wonder if if comes from my configuration that might be different: I built the image locally (it work fine there btw), then I uploaded the image on ec2 instance and ran it there making sure APPLICATION_URL is set.
The application showed up well, I configured the admin, but when I connect I get this ajax response `"{"status":"error","message":"Not Allowed"}"`

Any idea what I did wrong? 
I know ec2 restricts some ports and protocols by default, are there any other port than 80, 443 and 8334 for TCP I should open?
Thanks
